http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClientWireMockTest.java

```diff
@@ -618,6 +618,16 @@ public void testExceptionMessageChanged_WhenConnectionTimeoutErrorEncountered()
         customClient.close();
     }
 
+    @Test
+    public void createNettyClient_ReadWriteTimeoutCanBeZero() {
+        SdkAsyncHttpClient customClient = NettyNioAsyncHttpClient.builder()
+                .readTimeout(Duration.ZERO)
+                .writeTimeout(Duration.ZERO)
+                .build();
+
```

Comment:
Let's add `makeSimpleRequest(customClient);`  to make sure the requests can actually succeed. (Sorry, I should've mentioned earlier)